File: app/src/main/java/com/example/wanandroidsdk/http/WanHttpCallBack.java
package com.example.wanandroidsdk.http;


import com.example.wanandroidsdk.bean.WanDataBean;

import java.util.List;

/**
 * @auth: njb
 * @date: 2024/3/31 1:10
 * @desc: 请求回调
 */
public interface WanHttpCallBack {
    Boolean onResponse(List<WanDataBean> result);

    Boolean onFailure(String message);
}

File: app/src/main/java/com/example/wanandroidsdk/http/WanHttpCallBack.kt
package com.example.wanandroidsdk.http

import com.example.wanandroidsdk.bean.WanDataBean

/**
 * @auth: njb
 * @date: 2024/4/14 12:46
 * @desc: 描述
 */
interface WanHttpCallBack {
    fun onResponse(result: List<WanDataBean>): Boolean

    fun onFailure(message: String?): Boolean
}